SU2_CFD/include/numerics/elasticity/CFEAElasticity.hpp

```diff
@@ -183,16 +183,20 @@ class CFEAElasticity : public CNumerics {
    * \brief Compute VonMises stress from components Sxx Syy Sxy Szz Sxz Syz.
    */
   template<class T>
-  static su2double VonMisesStress(unsigned short nDim, const T& stress) {
+  static su2double VonMisesStress(unsigned short nDim, const T& stress, su2double Nu, bool isPlaneStrain) {
     if (nDim == 2) {
       su2double Sxx = stress[0], Syy = stress[1], Sxy = stress[2];
```

Comment:
If the 4th element of "stress" is Szz you don't need to pass Nu and isPlainStrain to this function.